test/malloc_bench.c

```diff
@@ -85,7 +85,7 @@ void randoms() {
           total_allocated += size;
         }
       } else {
-        if (calloc_ && USE_CALLOC) {
+        if (!calloc_ || !USE_CALLOC) {
```

Comment:
Keep the condition expressed in positives and flip the bodies instead?

What is `calloc_`  (with the trailing underscore)?  When would `USE_CALLOC` befined but not `calloc_`

Reply:
The benchmark has a global setting `USE_CALLOC`, and a random decision in each iteration as well. So it is checking if we allow calloc both globally and locally.

I'll reverse the condition.